samples/endpoint-jwt/README.md

```diff
@@ -5,40 +5,72 @@
 
 To understand the Akka concepts behind this example, see [Development Process](https://doc.akka.io/concepts/development-process.html) in the documentation.
 
-This project demonstrates the use of an Event Sourced Entity.
+## Developing
+
+This project demonstrates the use of JWT in a HTTP Endpoint.
 To understand more, read [JSON Web Tokens (JWT)](https://doc.akka.io/java/auth-with-jwts.html) in the documentation.
 
+## Building
+
 Use Maven to build your project:
 
 ```shell
 mvn compile
 ```
+## Running Locally
 
-
-When running an Akka service locally.
-
-To start your service locally, run:
+To start your Akka service locally, run:
 
 ```shell
 mvn compile exec:java
 ```
 
-This command will start your Akka service. With your Akka service running, the endpoint it's available at:
+## Exercising the service
+
+With your Akka service running, any defined endpoints should be available at `http://localhost:9000`.
 
 Run the command below, to test you can access your endpoint if you pass `iss`:`my-issuer` in the token. 
 Note the signature of the token is not being passed. Only the header and payloads are included. 
 More info in JWTs header, payload, and signature here: https://jwt.io/introduction.
+
 ```shell
 curl localhost:9000/hello --header "Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJteS1pc3N1ZXIifQ"
 ```
 
 Run the command below, to test you can NOT access your endpoint with any other `iss`, like for example `wrong-issuer`. 
 If interested, you can decode the token in https://jwt.io.
-```shell
 
+```shell
 curl localhost:9000/hello -i --header "Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ3cm9uZy1pc3N1ZXIifQ"
 ```
-Now if you `deploy` the service and `expose` it (write the output of the route below) 
```

Comment:
This was the reason I changed this. It just jumps into `deploy` without further explanations